strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/configuration/security/ldap/LdapAuthenticatorConfigurationController.java

```diff
@@ -55,7 +55,7 @@ public ResponseEntity getRolesMapping()
     {
         if (!springSecurityLdapInternalsSupplier.isLdapAuthenticationEnabled())
         {
-            return toResponseEntityError(LdapMessages.NOT_CONFIGURED, HttpStatus.BAD_REQUEST);
+            return toResponseEntityError(LdapConstants.LdapMessages.NOT_CONFIGURED, HttpStatus.BAD_REQUEST);
```

Comment:
* If there has to be a separate class for these messages, it should be public, not a nested class.
* Also, I'm not quite sure we need to have it this way. I understand -- it was made like this before... But at some point when we consider implementing internationalization, we'll have to revisit this as well.

Reply:
P.S. Just as a heads up - the `toResponseEntityError` should be replaced with the `getFailedAction` method from the `BaseController`. That will be part of another task. If you see this anywhere else during future tasks feel free to "refactor" it :)

Reply:
Do I separate this nested class then? Or I leave it like it is and we'll create a separate issue for this?

Reply:
Yes, please, extract the nested class into a public one.